src/test/java/com/google/sps/BusinessesServiceTest.java

```diff
@@ -0,0 +1,98 @@
+import java.util.List;
+import java.util.LinkedList;
+import java.util.Arrays;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.sps.data.BusinessesService;
+import com.google.sps.data.Listing;
+import com.google.sps.data.MapLocation;
+import static org.mockito.Mockito.*;
+
+@RunWith(JUnit4.class)
+public final class BusinessesServiceTest {
+
+  private final int MIN_LISTINGS_RETURNED = 20;
+  private final int MAX_LISTINGS_RETURNED = 60;
+  
+  // Address location in Owings Mills, MD
+  private final MapLocation TEST_MAP_LOCATION = new MapLocation(39.459836, -76.742536);
+  private final String TEST_PRODUCT = "pizza";
+
+  private final List<Listing> TEST_PASSED_LIST = new LinkedList<>();
```

Comment:
List of what exactly? Businesses? BigBusinesses? SmallBusiness?

Reply:
I changed the name to TEST_ALL_BUSINESSES which is the name of the parameter passed into the businesses service constructor - does that make more sense?